nxviz/plots.py

```diff
@@ -489,20 +489,29 @@ class CircosPlot(BasePlot):
     :param node_label_layout: which/whether (a) node layout is used,
         either 'rotation', 'numbers' or None
     :type node_label_layout: `string`
+    :type group_label_offset: `int`
```

Comment:
I think a float should work too! Also took a stab at writing down the intent of the kwarg.

```suggestion
    :param group_label_offset: how much to offset the group labels, so that they are
        not overlapping with node labels.
    :type group_label_offset: `float` or `int`
```